torchvision/transforms/v2/_auto_augment.py

```diff
@@ -698,38 +763,43 @@ def _unflatten_and_insert_image_or_video_and_bboxes(
 
     @staticmethod
     def _transform_image_or_video_in_bboxes(
-        fn: Callable[..., torch.Tensor], fn_kwrgs: dict, image: torch.Tensor, bboxes: torch.Tensor
+        fn: Callable[..., torch.Tensor], fn_kwrgs: dict, image: torch.Tensor, bboxes: datapoints.BoundingBox
     ) -> torch.Tensor:
         new_img = image.clone()
-        int_bboxes = bboxes.to(torch.long)
-        for bbox in int_bboxes:
+        xyxy_bboxes = F.convert_format_bounding_box(bboxes, new_format=datapoints.BoundingBoxFormat.XYXY)
+        for bbox in xyxy_bboxes:
```

Comment:
Nice catch about xyxy format, however boxes should be also dtype long as I committed previously:
```
        int_bboxes = bboxes.to(torch.long)
        for bbox in int_bboxes:
```